docs/solutions/sql_server_always_on_availability_groups/configuration.tf

```diff
@@ -0,0 +1,106 @@
+// vcn_dns_name is the outermost prefix in the <vcn_dns_name>.oraclevcn.com domain.
```

Comment:
I can't seem to comment on ConfigurationFile.ini since github thinks it's binary, but where is that file used? I'm not seeing it.

Reply:
Ah, right, I'm forgetting that these are all Windows instances, and we don't currently have a way of automating configuration in that case... so presumably your readme/doc will explain how to log in and set things up.

Reply:
That is correct. The .ini file is used to setup SQL on the windows instances and is not used by TF